ext/standard/basic_functions.stub.php

```diff
@@ -2045,6 +2087,7 @@ function sys_getloadavg(): array|false {}
 
 /**
  * @return object|array<string, mixed>|false
+ * @compile-time-eval
```

Comment:
```
========DIFF========
001+ php: /home/runner/work/php-src/php-src/ext/opcache/zend_persist_calc.c:150: zend_persist_zval_calc: Assertion `zval_get_type(&(*(z))) < IS_STRING' failed.
002+ Aborted (core dumped)
003+ 
004+ Termsig=6
001- object(stdClass)#1 (34) {
002-   ["browser_name_regex"]=>
003-   string(6) "~^.*$~"
004-   ["browser_name_pattern"]=>
005-   string(1) "*"
006-   ["browser"]=>
007-   string(15) "Default Browser"
008-   ["version"]=>
009-   string(1) "0"
010-   ["majorver"]=>
011-   string(1) "0"
012-   ["minorver"]=>
013-   string(1) "0"
014-   ["platform"]=>
015-   string(7) "unknown"
016-   ["alpha"]=>
017-   string(0) ""
018-   ["beta"]=>
019-   string(0) ""
020-   ["win16"]=>
021-   string(0) ""
022-   ["win32"]=>
023-   string(0) ""
024-   ["win64"]=>
025-   string(0) ""
026-   ["frames"]=>
027-   string(1) "1"
028-   ["iframes"]=>
029-   string(0) ""
030-   ["tables"]=>
031-   string(1) "1"
032-   ["cookies"]=>
033-   string(0) ""
034-   ["backgroundsounds"]=>
035-   string(0) ""
036-   ["authenticodeupdate"]=>
037-   string(1) "0"
038-   ["cdf"]=>
039-   string(0) ""
040-   ["vbscript"]=>
041-   string(0) ""
042-   ["javaapplets"]=>
043-   string(0) ""
044-   ["javascript"]=>
045-   string(0) ""
046-   ["activexcontrols"]=>
047-   string(0) ""
048-   ["stripper"]=>
049-   string(0) ""
050-   ["isbanned"]=>
051-   string(0) ""
052-   ["wap"]=>
053-   string(0) ""
054-   ["ismobiledevice"]=>
055-   string(0) ""
056-   ["issyndicationreader"]=>
057-   string(0) ""
058-   ["crawler"]=>
059-   string(0) ""
060-   ["css"]=>
061-   string(1) "0"
062-   ["cssversion"]=>
063-   string(1) "0"
064-   ["supportscss"]=>
065-   string(0) ""
066-   ["aol"]=>
067-   string(0) ""
068-   ["aolversion"]=>
069-   string(1) "0"
070- }
========DONE========
FAIL Bug #77338: get_browser with empty string [ext/standard/tests/misc/bug77338.phpt] 
```
is the assertion in the test failure correct?

Reply:
Yes, functions which return objects cannot be evaluated.